### PR TITLE
[uss_qualifier] Organize DSS tests

### PR DESCRIPTION
**Pull request by BenjaminPelletier**
This PR establishes a new test suite to validate the InterUSS DSS implementation which implements DSS requirements across three standards (though we only currently have dedicated uss_qualifier DSS tests for F3411-19 and F3411-22a) and makes the resource definitions in the configuration explicit rather than $ref'd so the full content is easily accessible.

It also regroups the DSS testing for F3411-19 and F3411-22a to run the interoperability test for all participants acting as primary DSS -- this allows a number of requirements to be verified for all participants that previously weren't checked when only one participant's DSS acted as primary DSS.  In order to accomplish this, an optional resource specifying all the other DSS instances is declared in each of the standard's dss_probing test suites and used for the interoperability scenario if present.  To support cases where someone wants to test only a single DSS instance, support is added for skipping test suite actions that don't have the necessary resources specified (see suite README.md).

Finally, some test suite documentation_urls are populated and an off-by-one error corrected.

### CODE REVIEW

File: monitoring/uss_qualifier/suites/astm/netrid/f3411_19.yaml [hunk 1]
```diff
@@ -28,11 +31,6 @@ actions:
       scenario_type: scenarios.astm.netrid.v19.dss.CRDBAccess
       resources: {}
       on_failure: Continue
```
Comment (mickmis) [inline monitoring/uss_qualifier/suites/astm/netrid/f3411_19.yaml:33]: Shouldn't the `CRDBAccess` scenario also be transferred to the `dss_probing` suite?

File: monitoring/uss_qualifier/suites/astm/netrid/f3411_22a.yaml [hunk 2]
```diff
@@ -30,11 +31,6 @@ actions:
       scenario_type: scenarios.astm.netrid.v22a.dss.CRDBAccess
       resources: {}
       on_failure: Continue
```
Comment (mickmis) [inline monitoring/uss_qualifier/suites/astm/netrid/f3411_22a.yaml:33]: (same as above)
Shouldn't the `CRDBAccess` scenario also be transferred to the `dss_probing` suite?

File: monitoring/uss_qualifier/suites/astm/netrid/f3411_19/dss_probing.yaml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 name: DSS instance probing for ASTM NetRID F3411-19
```
Comment (mickmis) [inline monitoring/uss_qualifier/suites/astm/netrid/f3411_19/dss_probing.yaml:1]: nit: consistency
```suggestion
name: DSS testing for ASTM NetRID F3411-19
```

File: monitoring/uss_qualifier/suites/interuss/dss/all_tests.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        dss_instance_id: dss
+      on_failure: Continue
+  - test_scenario:
+      scenario_type: scenarios.astm.netrid.v19.dss.CRDBAccess
+      resources: {}
+      on_failure: Continue
+  - test_scenario:
+      scenario_type: scenarios.astm.netrid.v22a.dss.CRDBAccess
+      resources: {}
+      on_failure: Continue
```
Comment (mickmis) [inline monitoring/uss_qualifier/suites/interuss/dss/all_tests.yaml:53]: Analog to 2 other comments: Shouldn't the `CRDBAccess` scenario also be transferred to the `dss_probing` suite?

File: monitoring/uss_qualifier/resources/astm/f3411/dss.py [hunk 5]
```diff
@@ -73,6 +73,15 @@ def __init__(
         if local_debug is not None:
             self.local_debug = local_debug
 
+    def is_same_as(self, other: DSSInstance) -> bool:
```
Comment (mickmis) [inline monitoring/uss_qualifier/resources/astm/f3411/dss.py:76]: nit: Shouldn't comparing the participant IDs be enough? It looks like if there are different DSSs declared with the same participant ID, of the same DSSs declared multiple times with different configuration, it should error out before.

**Review by mickmis — APPROVED** [review]
Side note: depending on which gets merged first between this PR and #204, the `service_area` resource in `dss_probing.yaml` will need to be updated.

Side note bis: this test framework is really becoming powerful and in a mostly generic way, have you ever considered factoring it out in a project of its own at some point? :)

File: monitoring/uss_qualifier/suites/astm/netrid/f3411_19.yaml (hunk 1 above)
Comment (BenjaminPelletier) [reply monitoring/uss_qualifier/suites/astm/netrid/f3411_19.yaml:33]: Yes, I intended to do this but apparently didn't do it successfully -- thanks, fixed (locally).

File: monitoring/uss_qualifier/suites/astm/netrid/f3411_22a.yaml (hunk 2 above)
Comment (BenjaminPelletier) [reply monitoring/uss_qualifier/suites/astm/netrid/f3411_22a.yaml:33]: Yes, thanks -- fixed (locally)

File: monitoring/uss_qualifier/suites/astm/netrid/f3411_19/dss_probing.yaml (hunk 3 above)
Comment (BenjaminPelletier) [reply monitoring/uss_qualifier/suites/astm/netrid/f3411_19/dss_probing.yaml:1]: Thanks, harmonized (locally)

File: monitoring/uss_qualifier/suites/interuss/dss/all_tests.yaml (hunk 4 above)
Comment (BenjaminPelletier) [reply monitoring/uss_qualifier/suites/interuss/dss/all_tests.yaml:53]: Yes, fixed -- thanks

File: monitoring/uss_qualifier/resources/astm/f3411/dss.py (hunk 5 above)
Comment (BenjaminPelletier) [reply monitoring/uss_qualifier/resources/astm/f3411/dss.py:76]: It should be relatively common for a single participant to provide multiple instances -- InterUSS's DSS needs an odd number of instances and some redundancy is good practice, so the baseline starting instance count for a normal deployment is 3.  If there are fewer than 3 USSs in the ecosystem who want to help host the DSS pool, then one participant will necessarily provide multiple instances.  I think it should be fine to provide multiple DSS instances from the same participant (both conceptually and in the current implementation)?  If not, I'll follow up with a fix in another PR.

Comment (mickmis) [reply monitoring/uss_qualifier/resources/astm/f3411/dss.py:76]: Ohh I didn't think at all about the 1 participant having multiple instances, that makes sense yes.

**Pull request merged**